roles/audit_storage/molecule/default/converge.yml

```diff
@@ -6,4 +6,6 @@
       vars:
         activemq_version: "{{ dependencies_version.activemq }}"
     - role: elasticsearch
+      vars:
+        elasticsearch_major_version: "{{ '8.x' if acs.version is version('25.0', 'ge') else '7.x' }}"
```

Comment:
not sure if it's really worth having this logic for molecule, but it would prevent us to forget switching to 8.x by default, so maybe we can still drop it later